crates/db_schema/src/utils/series.rs

```diff
@@ -0,0 +1,81 @@
+use diesel::{
+  dsl,
+  expression::{is_aggregate, ValidGrouping},
+  pg::Pg,
+  query_builder::{AsQuery, AstPass, QueryFragment},
+  result::Error,
+  sql_types,
+  AppearsOnTable,
+  Expression,
+  Insertable,
+  SelectableExpression,
+};
+
+#[derive(QueryId)]
```

Comment:
What is all this for, and what is it doing?

Reply:
I added documentation on `ValuesFromSeries` to explain it

Reply:
Move this into db_perf crate as its only used there.

Reply:
Done